toolkits/global/packages/global-forms/README.md

```diff
@@ -48,13 +51,13 @@ Fieldsets are used to group fields itemised under their `fields` property. If yo
         {
             "fields": [
                 {
-                    "template": "text",
+                    "template": "globalFormText",
                     "label": "Your name",
                     "id": "your-name",
                     "name": "your-name"
                 },
                 {
-                    "template": "text",
+                    "template": "globalFormText",
```

Comment:
does this need to be `globalFormEmail` or does it not matter?

Reply:
similarly with all other instances? I guess it would be know what it is doing depending on the label, id, name in the `json` to negate each field needing it's own `globalFormThing`

Reply:
@stuRobson this determines which type of field it is (text,email,radios,select etc). The verbose name is because some templates may need to be registered globally and there may already be something in the global scope simply called `text`.

Reply:
> does this need to be globalFormEmail or does it not matter?

Ah, I see what you mean. Yeah, that would be better.

Reply:
👍🏼